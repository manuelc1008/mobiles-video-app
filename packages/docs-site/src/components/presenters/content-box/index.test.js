import React from 'react'
import renderer from 'react-test-renderer'

import ContentBox from './index'

/* eslint-disable react/no-children-prop */

describe('ContentBox', () => {
  it('snapshot: has same HTML structure', () => {
    const tree = renderer
      .create(<ContentBox children={['Content goes here']} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
