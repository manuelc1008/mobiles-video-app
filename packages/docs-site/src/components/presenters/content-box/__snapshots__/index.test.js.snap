// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentBox snapshot: has same HTML structure 1`] = `
<section
  className="content-box "
>
  Content goes here
</section>
`;
